js/MediaStreamRenderer.js

```diff
@@ -257,7 +261,7 @@ MediaStreamRenderer.prototype.refresh = function () {
 			if (elementRatio >= videoRatio) {
 				videoViewWidth = elementWidth;
 				videoViewHeight = videoViewWidth / videoRatio;
-			// The element has lower width/height ratio than the video.
+				// The element has lower width/height ratio than the video.
```

Comment:
Align on if ?

Reply:
Yeah, there are a number of these comments that ended up in an awkward position.  Prettier indents them properly to match the code block in which they reside.  What may make more sense is to move them into the block that they are describing.  In this case, into the `} else if (elementRatio < videoRatio) {`.  If you agree that that's the better place for them, i'll go through and move each of them.

Reply:
Yes pls